File: Gixor-Backend/Gixor-Api/Gixor-Api-GitHub/src/main/java/com/cpy3f2/Gixor/Domain/DTO/IssueDTO.java
package com.cpy3f2.Gixor.Domain.DTO;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

/**
 * @author : simon
 * @description :
 * @last : 2024-11-04 00:48
 * Copyright (c) 2024. 保留所有权利。
 * @since : 2024-11-04 00:48
 */
@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
@JsonIgnoreProperties(ignoreUnknown = true)
public class IssueDTO {
    private String title;
    private String body;
    private List<String> assignees;

    private Long milestone;

    private List<String> labels;
    private String state;
    @JsonProperty("state_reason")
    private String stateReason;
}

File: Gixor-Mobile/app/src/main/java/com/cpy3f2/gixor_mobile/model/entity/IssueDTO.kt
package com.cpy3f2.gixor_mobile.model.entity


data class IssueDTO(
    val title: String? = null,
    val body: String? = null,

    //指派人
    val assignees: List<String>? = null,

    val milestone: Long? = null,

    //标签
    val labels: List<String>? = null,
    val state: String? = null,

    val stateReason: String? = null
)